index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" >
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css" >
+    <script type="text/javascript" src="./main.js"></script>
+  </head>
+  <body>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="section">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+            required
+          >
+        </div>
+        <div class="section">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+            required
+          >
+        </div>
+        <div class="section">
+          <label for="age">How old are You?</label>
+          <input
+            type="number"
+            name="age"
+            id="age"
+            min="1"
+            max="100"
+            value="12"
+            required
+          >
+        </div>
+        <div class="section">
+          <label for="dob">Full date of birth:</label>
+          <input
+            type="date"
+            name="dob"
+            id="dob"
+            required
+          >
+        </div>
+        <div class="section">
+          <label for="agreement">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="agreement"
+            id="agreement"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div class="section">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+          >
+        </div>
+        <div class="section">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="8"
+            maxlength="10"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting facts about you</legend>
+        <div class="section">
+          <label for="radio">Do you love cats?</label>
+          <input type="radio"
+            name="radio"
+            id="yes"
+            value="Yes"
+          >
```

Comment:
each attributes should be on new line. Check your code please and fix according this rule

```suggestion
          <input 
            type="radio"
            name="radio"
            id="yes"
            value="Yes"
          >
```

Reply:
I cannot deploy second link